src/components/RefineSearch/RefineSearch.tsx

```diff
@@ -20,7 +21,18 @@ import {
   getQueryWithoutFilters,
 } from "../../utils/refineSearchUtils"
 import type { Aggregation } from "../../types/filterTypes"
+import DateForm from "../DateForm"
+import { useDateForm } from "../SearchFilters/DateForm"
 
+const debounceInterval = 60
```

Comment:
Consider making 20 or 60 a default in the hook so you don't always have to pass down a value.

Reply:
done!